### PR TITLE
Cleanup naming of methods and variables

### PR DESCRIPTION
**Pull request by olilarkin**
This PR goes some way to cleaning up the plug-in code, renaming "NAM" methods/variables to "Model" and a few other things. see #267

### CODE REVIEW

**Comment by olilarkin** [comment]
PR updated

**Comment by olilarkin** [comment]
how about it @sdatkinson? I think it will make the code a lot nicer. :-)

will keep on having to redo this PR if it sits around whilst other changes get merged

File: NeuralAmpModeler/NeuralAmpModeler.cpp [hunk 1]
```diff
@@ -51,36 +51,20 @@ const IVStyle style =
 
 NeuralAmpModeler::NeuralAmpModeler(const InstanceInfo& info)
 : Plugin(info, MakeConfig(kNumParams, kNumPresets))
-, mInputPointers(nullptr)
```
Comment (sdatkinson) [inline NeuralAmpModeler/NeuralAmpModeler.cpp:54]: [Sorry for incomplete review, short on time] isn't it potentially problematic not to initialize these?

Comment (olilarkin) [reply NeuralAmpModeler/NeuralAmpModeler.cpp:54]: mInputPointers is initialised inline instead

**Review by sdatkinson — APPROVED** [review]
LGTM! 👍🏻

**Pull request merged**